import-export-cli/git/gitUtils.go

```diff
@@ -629,8 +632,9 @@ func DeployChangedFiles(accessToken, environment string) map[string][]*params.Pr
 }
 
 // Create 'vcs.yaml' in the repository root folder with a unique id (uuid) for the repository.
-//  If the value of force is false, and the file is already created, gives an error.
-//  If the value of force is true, It will reinitialize the file even if it already exists.
+//
+//	If the value of force is false, and the file is already created, gives an error.
+//	If the value of force is true, It will reinitialize the file even if it already exists.
```

Comment:
Seems this change is not required. Please check.

Reply:
Resolved by https://github.com/wso2/product-apim-tooling/pull/1027/commits/bf810875b56eb0bb7f3328899408af14d8d243e8